src/function.ts

```diff
@@ -20,6 +21,7 @@ interface SourceFile {
   srcDir: string
   srcPath: string
   stat: Stats
+  inSourceConfig?: ISCValues
```

Comment:
[dust] Can we maintain alphabetical order of properties?

Reply:
outdated by https://github.com/netlify/zip-it-and-ship-it/pull/899/commits/15c1cadfe33854a8bee6810cd66923f22f22a089